plugin/trino-snowflake/src/test/java/io/trino/plugin/snowflake/TestSnowflakeConnectorTest.java

```diff
@@ -247,6 +248,31 @@ public void testLimitPushdown()
         abort("TODO");
     }
 
+    @Test
+    @Disabled
+    @Override
+    public void testCommentTable()
+    {
+        // This connector does not support setting table comments
```

Comment:
The test verifies the expected failure message. We shouldn't skip tests like this. It indicates hasBehavior is wrong.

Reply:
Added it into hasBehavior.